build_tools/configure_ci.py

```diff
@@ -349,7 +349,8 @@ def main(base_args, build_families, test_families):
         build_linux_target_output = []
         build_windows_target_output = []
 
-        # If this is a (main push or PR trigger) and enable_build_jobs is False, skip the tests since there is no build to use.
+        # If this enable_build_jobs flag is set to false and the trigger is either a main push or pull request,
```

Comment:
```suggestion
        # If the `enable_build_jobs` flag is set to false and the trigger is either a push to main or pull request,
```
Thanks for clarifying!